setup.py

```diff
@@ -31,8 +31,11 @@
     packages=['pyoozie'],
     install_requires=[
         'enum34>=0.9.23',
-        'yattag>=1.7.2',
+        'requests>=2.12.3',
         'setuptools>=0.9',
+        'six',
+        'untangle',
```

Comment:
Apparently both undecorated packages are over a year old in the their current rev (4yrs for untangle!); I'll tag that as a minimum.